Biospe_Custom_QC_Accession_Dup.Rmd

```diff
@@ -0,0 +1,1021 @@
+FROM `nih-nci-dceg-connect-prod-6d04.FlatConnect.participants_JP` where Connect_ID IS NOT NULL"
+
+parts_table <- bq_project_query(project, parts)
+parts_data <- bq_table_download(parts_table, bigint = "integer64")
+
+bio_table <- bq_project_query(project, bio)
+bio_data <- bq_table_download(bio_table, bigint = "integer64")
+
+bio_data$Connect_ID <- as.numeric(bio_data$Connect_ID)
+parts_data$Connect_ID <- as.numeric(parts_data$Connect_ID)
+
+
+
+#bioqc_join= left_join(parts_data, bio_data, by="Connect_ID") 
+
+#bioqc <- as_tibble(bioqc_join)
+
+bioqc= left_join(bio_data, parts_data,  by=c("Connect_ID", "d_827220437") )
+
+knitr::opts_chunk$set(comment = NA)
+
+
+
+
+
+bioqc <- bioqc %>%  mutate(Site=case_when(d_827220437==531629870 ~ "HealthPartners",
+                                                 d_827220437==548392715 ~ "Henry Ford Health System",
+                                                 d_827220437==303349821 ~ "Marshfield Clinical Health System",
+                                                 d_827220437==657167265 ~ "Sanford Health",
+                                                 d_827220437==809703864 ~ "University of Chicago Medicine",
+                                                d_827220437==125001209 ~ "Kaiser Permanente Colorado",
+                                                 d_827220437==327912200 ~ "Kaiser Permanente Georgia",
+                                                 d_827220437==300267574 ~ "Kaiser Permanente Hawaii",
+                                                 d_827220437==452412599 ~ "Kaiser Permanente Northwest"))
+
+```
+
+
+```{r, echo=FALSE, warning=FALSE, message=FALSE}
+
+
+#### IN A SEPARATE REPORT
+
+
+### HAWAII ONLY: They're sending one extra number
+
+#qc1 <- bioqc %>%  filter(d_827220437==300267574)
+
+#Blood
+qc1_blood <- bioqc %>%  filter(str_sub(d_173836415_d_266600170_d_341570479,1,11)!=str_sub(d_646899796_integer,1,11)) 
+
+qc1_blood <- qc1_blood %>% select(Site, Connect_ID, d_173836415_d_266600170_d_341570479, d_646899796_integer, d_556788178 ) %>%  sort_by(Site)
+
+
+# qc1_blood %>%  gt() %>%  
+#   tab_header(title = md("KPHI Blood Accession ID Errors")) %>% 
+#   cols_label(Connect_ID = md("**Connect ID**"), d_173836415_d_266600170_d_341570479 = md("**KPHI Accession ID**"), d_646899796_integer= md("**Expected Accession ID**"))
+
+colnames(qc1_blood) <- c("Site", "Connect ID", "KP Accession ID" , "Expected Accession ID", "Collection Date/Time")
+#knitr::kable(qc1_blood)
+
+write.csv(qc1_blood,paste(local_drive, "Mismatched_Blood_Accession_IDs",currentDate,".csv",sep="_"),row.names = F,na="")
```

Comment:
please add the box folder that you would like this to go to. Something like this:

```
boxfolder = "12334567812"
write.csv(qc1_blood,paste(local_drive, "Mismatched_Blood_Accession_IDs",currentDate,"_boxfolder_",boxfolder,".csv",sep="_"),row.names = F,na="")
```